src/SAREhub/Client/Processor/LogProcessor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace SAREhub\Client\Processor;
+
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerInterface;
+use SAREhub\Client\Message\Exchange;
+
+class LogProcessor implements Processor,LoggerAwareInterface
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    public function process(Exchange $exchange)
+    {
+        $this->logger->info($exchange);
```

Comment:
Add setter and getter for log level and use generic log method
$this->logger->log($level, $message, array $context = ["exchange" => $exchange]);
Use id(IdAware) for message parameter